docs/_documentations/mdt-eclipse-buildproject.md

```diff
@@ -10,9 +10,9 @@ order: 40
 parent: mdteclipseoverview
 ---
 
-# Building Codewind projects
+# Codewind build logs
 
-When auto build is enabled for a project, Codewind for Eclipse detects when you make a change and starts a build automatically. If you have disabled auto build for the project, you can start a build manually when you have made a change or a set of changes:
+If you enable auto build for a project, Codewind for Eclipse, Eclipse Che, and Visual Studio Code (VS Code) detects when you make a change and starts a build automatically. If you have disabled auto build for a project, you can start a build manually when you have made a change or a set of changes:
```

Comment:
- I think it's OK to refer to "Codewind" as just "Codewind" instead of "Codewind for Eclipse."
- Since you have a plural subject (all three of those programs), you should make the verbs "detect" and "start" to match. 
- The second sentence should also be present tense: "If you disable auto build..." and "...when you make a change..."

Reply:
I read the first sentence a little differently. I read it as "Codewind [for three IDEs] for Eclipse, Eclipse Che, and VS Studio" and essentially it's saying, "Codewind detects when you make a change and starts to build..." So, it's not the three IDEs that are the subject, it's Codewind.

Reply:
Would it be OK to omit mention of the specific IDEs and say something like, "Codewind detects when you make..."? Because this file is an Eclipse file, I think the users would assume it's the Eclipse IDE, anyway.

Reply:
Sure thing, I can and shall do that. Good suggestion, thanks!